modules/general/external-dns/main.tf

```diff
@@ -95,18 +93,24 @@ resource "helm_release" "external_dns" {
     type  = "string"
   }
 
+  set {
+    name  = "triggerLoopOnEvent"
```

Comment:
no need as per [here](https://github.com/kubernetes-sigs/external-dns/issues/2636)

Reply:
that is needed, so the external dns is triggered by event, not loop. So as to reduce the API throttle. https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/aws.md#throttling

Reply:
we monitor service and ingress, not crd